config/initializers/sucker_punch.rb

```diff
@@ -0,0 +1,2 @@
+require 'sucker_punch/async_syntax'
+SuckerPunch.logger = Logger.new("#{Rails.root}/log/sucker_punch.log")
```

Comment:
I'm not sure about this one. People configure their loggers in various ways. I think a better default would be to use Rails.logger here. That would make it more likely that the logging configuration people are already using would work.